hbw/app/models/hbw/activiti/adapter.rb

```diff
@@ -15,6 +15,13 @@ def initialize(entity_code_key: , activiti_api: )
       def users
         HBW::BPMUser.fetch_all
       end
+      
+      def user_exist?(user_email)
+        user = HBW::BPMUser.with_connection(api) do
+          HBW::BPMUser.fetch(user_email)
+        end
+        user.present?
```

Comment:
I'd prefer to use native ruby's `nil?` like this `!user.nil?`. AS' `present?` is an inversion of `blank?` and `blank?` does too much, and we don't like it :)